lua/nvim-autopairs/rules/basic.lua

```diff
@@ -29,8 +29,10 @@ local function setup(opt)
         Rule('"""', '"""', { 'python', 'elixir' }),
         basic("'", "'")
             :with_pair(cond.not_before_regex_check("%w"))
-            :with_pair(cond.not_filetypes({"rust"}))
-        ,
+            :with_pair(cond.not_filetypes({"rust"})),
+        basic("'", "'", "rust")
+            :with_pair(cond.not_before_regex_check("[%w<&]")),
```

Comment:
I'm not very familiar with lua but should there be a comma at the end of this line? It doesn't seem to match the patterns elsewhere in this source file.

Reply:
You are right. It seems I can't type correctly 😅